### DISCUSSION TITLE
Copilot doesn't answer questions in the context of the chat history

### DISCUSSION THREAD

**Discussion by james-berkheimer** [community/community#128599] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

When I am working on a coding problem I like to use chat.  But I am constantly having to re-enter code into the chat to provide Copilot with context.  I am a bit surprised that it has no awareness of the chat history when answering coding questions.  ChatGPT seems to do this just fine.  So are we ever going to get this in Copilot?  Seems like a no-brainer.

![image](https://github.com/community/community/assets/5049386/691882cf-9d84-43f4-adc1-a5ea60f9562a)

**Comment by EdgarJP006** [comment]
I guess it is because of the amount of code that can be in a project, I know that it acquires part of the context by you placing the cursor over the portion of code or selecting that portion and in the chat you ask the question and from there it gets the context, but at the level as ChatGPT does it, I think it would not work the same.
